tests/phpunit/tests/blocks/register.php

```diff
@@ -234,6 +233,52 @@ public function test_missing_asset_file_register_block_script_handle() {
 		$this->assertFalse( $result );
 	}
 
+	/**
+	 * @ticket 57234
+	 */
+	public function test_missing_asset_php_for_traditionally_registered_block() {
+		$metadata = array(
+			'file'   => DIR_TESTDATA . '/blocks/notice/block.json',
+			'name'   => 'unit-test/traditional-block',
+			'script' => 'file:./blocks/notice/block.js',
+		);
+		$result   = register_block_script_handle( $metadata, 'script' );
+
+		// Assert that the result is false due to missing .asset.php
+		$this->assertFalse( $result );
```

Comment:
What’s the reason that it returns `false`? Shouldn't it return the generated handle name instead?